portworx-replication-set/step6.md

```diff
@@ -0,0 +1 @@
+>>Q11. Mount the volume 'shared_volume' on node03 at the directory /var/lib/osd/mounts/node03<<
```

Comment:
I assume this is using `pxctl host attach` then `pxctl host mount`? Might be worth saying `"with pxctl`

Reply:
This file is not used in the index.json for this lab, can be ignored.